### ISSUE TITLE
Specify Output Operator Name and Use As Output Directory

### ISSUE THREAD

**Issue by aneeshkp** [redhat-nfvpe/helm2go-operator-sdk#13]
**Describe the bug**
 `go run main.go convert --api-version=apps.example.com/v1aplha1 --helm-chart test/bitcoind/ --kind=Test --output-dir Test-Operator`

No way to specify the output operator name. Expects output directory to exist.

**To Reproduce**
`go run main.go convert --api-version=apps.example.com/v1aplha1 --helm-chart test/bitcoind/ --kind=Test --output-dir Test-Operator`

**Expected behavior**
Instead of setting `--out-dir` , ask for the operator name and create a directory.

**Screenshots**
```shell
go run main.go convert --api-version=apps.example.com/v1aplha1 --helm-chart test/bitcoind/ --kind=Test --output-dir Test-Operator
[2019-07-02 13:49:50] 🤠 Converting Existing Helm Chart test/bitcoind/ to Go Operator!
[2019-07-02 13:49:50] Resource Type *v1beta2.Deployment is Unsupported! Update API Version to Continue.

[2019-07-02 13:49:50] Resource Type *v1.PersistentVolumeClaim is Unsupported! Update API Version to Continue.

[2019-07-02 13:49:50] open Test-Operator/pkg/controller/test/test_controller.go: no such file or directory

false
[2019-07-02 13:49:50] 🤠 Go Operator Can Be Found At: Test-Operator         
```

**Comment by sjakati98** [comment]
Addressed in 272160e70250dd34de0b9b93a419812933998378

**Comment by aneeshkp** [comment]
Fixed
